pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -19,6 +19,8 @@ import (
 	"fmt"
 	"time"
 
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
```

Comment:
Do we need to alias this? We don't have any other packages named meta imported here?

Reply:
We don't _need_ to, I just thought having a consistent package name would be nice to have.

Reply:
gotcha - maybe we should keep our own packages with the name that we give them, and alias any externally conflicting packages? 

I'm guessing this is conflicting with the k8s meta package, which is normally imported as `metav1` IIRC?

Reply:
Ya that's a good idea, but for Elasticsearch we have multiple packages of our own. I wanted to keep the original name for the package that actually does stuff with elasticsearch, and alias the rendering package.